src/EA.Weee.Web/Areas/Scheme/ViewModels/ManageEvidenceNotes/ManageEvidenceNoteSchemeViewModel.cs

```diff
@@ -5,7 +5,7 @@
     using Aatf.ViewModels;
     using Web.ViewModels.Shared;
 
-    public abstract class ManageEvidenceNoteViewModel : IEvidenceNoteRowViewModel
+    public abstract class ManageManageEvidenceNoteSchemeViewModel : IManageEvidenceViewModel
```

Comment:
ManageManage?
Typo or just the quirk of naming convention?

Reply:
Updated...